### PR TITLE
Correct argument order to findDeployment()

### LINKED ISSUE #906: showLogs errs when deployment exists
Unable to call `showLogs()` without error. Deploy a Shiny application to shinyapps.io and then request its logs.

```
> rsconnect::showLogs(account = "aronatkins", server = "shinyapps.io")
Error in `rsconnect::showLogs()`:
! Couldn't find any deployments matching supplied criteria.
Run `rlang::last_trace()` to see where the error occurred.
> rlang::last_trace()
<error/rlang_error>
Error in `rsconnect::showLogs()`:
! Couldn't find any deployments matching supplied criteria.
---
Backtrace:
    ▆
 1. └─rsconnect::showLogs(account = "aronatkins", server = "shinyapps.io")
Run rlang::last_trace(drop = FALSE) to see 3 hidden frames.
> rlang::last_trace(drop = FALSE)
<error/rlang_error>
Error in `rsconnect::showLogs()`:
! Couldn't find any deployments matching supplied criteria.
---
Backtrace:
    ▆
 1. └─rsconnect::showLogs(account = "aronatkins", server = "shinyapps.io")
 2.   └─rsconnect:::findDeployment(appPath, appName, account, server)
 3.     └─cli::cli_abort(...)
 4.       └─rlang::abort(...)
```

The same problem happens when providing the name:

```
> rsconnect::showLogs(appName = "newshiny", account = "aronatkins", server = "shinyapps.io")
Error in `rsconnect::showLogs()`:
! Couldn't find any deployments matching supplied criteria.
Run `rlang::last_trace()` to see where the error occurred.
```

```
> rsconnect:::deployments()
      name    title   username    account       server                     hostUrl   appId bundleId
1 newshiny newshiny aronatkins aronatkins  posit.cloud https://api.shinyapps.io/v1 6184418  7432949
2 newshiny newshiny aronatkins aronatkins shinyapps.io https://api.shinyapps.io/v1 9425015  7432957
                                        url envVars version asMultiple asStatic
1       https://posit.cloud/content/6184418               1      FALSE    FALSE
2 https://aronatkins.shinyapps.io/newshiny/               1      FALSE    FALSE
                                    deploymentFile
1  ./rsconnect/posit.cloud/aronatkins/newshiny.dcf
2 ./rsconnect/shinyapps.io/aronatkins/newshiny.dcf
```

### PR DESCRIPTION
**Pull request by hadley**
Fixes #906. (I did not test it, but I assume that this is the root cause of the problem)

### CODE REVIEW

File: R/applications.R [hunk 1]
```diff
@@ -218,7 +218,12 @@ showLogs <- function(appPath = getwd(), appFile = NULL, appName = NULL,
                      account = NULL, server = NULL, entries = 50, streaming = FALSE) {
 
   # determine the log target and target account info
-  target <- findDeployment(appPath, appName, account, server)
+  target <- findDeployment(
```
Comment (aronatkins) [inline R/applications.R:221]: I'm not seeing this as sufficient..

```
> target <- rsconnect:::findDeployment(appPath = getwd(), appName = "newshiny", account = "aronatkins", server = "shinyapps.io")
> is.null(target)
[1] FALSE
> accountDetails <- rsconnect:::accountInfo(target$account, target$server)
> client <- rsconnect:::clientForAccount(accountDetails)
> application <- rsconnect:::getAppByName(client, accountDetails, target$appName)
> is.null(application)
[1] TRUE
```

It looks like we have `target$name` but not `target$appName`...

Comment (aronatkins) [reply R/applications.R:221]: Using `name` instead of `appName` appears to give me a valid application.

Comment (aronatkins) [reply R/applications.R:221]: Here, we are using a bare deployment object, which has its name as `name`. This is different than the `deployApp` paths, which call `deploymentTarget` and rewrite the deployment `name` to `appName`.

Comment (hadley) [reply R/applications.R:221]: Sigh, I'll guess I'll try testing it then 😄

**Review by aronatkins — APPROVED** [review]
With 6379c9335dd5239294e7f74ea93c3ef1e2ffb298, I can `rsconnect::showLogs` against a shinyapps.io deployment.

**Pull request merged**